torch/onnx/utils.py

```diff
@@ -174,16 +174,15 @@ def setup_onnx_logging(verbose: bool):
 
 @contextlib.contextmanager
 @_beartype.beartype
-def _setup_onnx_diagnostic():
+def _create_diagnostic_context():
```

Comment:
Can this function live outside of utils and be moved to a relavant module in diagnostics?

Reply:
Let's keep it simple and put it in _diagnostics.py then. Renaming to `create_diagnostic_context_for_export` or `create_export_diagnostic_context`, wdyt?

Reply:
`create_export_diagnostic_context` sounds about right